### PR TITLE
Widget header

### PR DESCRIPTION
**Pull request by menor**
have some static components working with fake server data

### CODE REVIEW

File: app/components/App.jsx [hunk 1]
```diff
@@ -1,6 +1,29 @@
 import React from 'react';
+import WidgetHeader from './WidgetHeader';
+import WidgetMessage from './WidgetMessage';
+
+const SERVER_DATA = {
+  message: 'El torito les desea felices pascuas',
+  facilityName: 'El Torito',
+  reservationCovers: 2,
+  reservationDate: new Date()
+};
```
Comment (neocortex) [inline app/components/App.jsx]: This shouldn't probably go here.

File: app/components/App.jsx [hunk 2]
```diff
@@ -1,6 +1,29 @@
 import React from 'react';
+import WidgetHeader from './WidgetHeader';
+import WidgetMessage from './WidgetMessage';
+
+const SERVER_DATA = {
```
Comment (neocortex) [inline app/components/App.jsx]: This shouldn't probably go here.

**Comment by neocortex** [comment]
Looks good, but I wouldn't put the server data constant in there. Maybe rather create an example mock-function outside to retrieve that test-data?

File: app/components/App.jsx (hunk 2 above)
Comment (menor) [inline app/components/App.jsx]: You're right it should go in a different store, but I'm faking it for now to have it running, and I'll deal with server calls later.

**Pull request merged**